index.yaml

```diff
@@ -1309,6 +1309,119 @@ entries:
     urls:
     - assets/rancher-gatekeeper/rancher-gatekeeper-crd-3.1.100.tgz
     version: 3.1.100
+  rancher-gke-operator:
+  - annotations:
+      catalog.cattle.io/auto-install: rancher-gke-operator-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: cattle-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/provides-gvr: gkeclusterconfigs.gke.cattle.io/v1
+      catalog.cattle.io/rancher-version: 2.0.1-rc1
+      catalog.cattle.io/release-name: rancher-gke-operator
+      catalog.cattle.io/scope: management
+    apiVersion: v2
+    appVersion: 2.0.1-rc1
+    created: "2021-05-18T16:21:52.95982-07:00"
+    description: A Helm chart for provisioning GKE clusters
+    digest: 9ccb55efd79d1c5f608d349837ca978ddb590f1e639528dd62e7ce12cf2e8955
+    home: https://github.com/rancher/gke-operator
+    name: rancher-gke-operator
+    sources:
+    - https://github.com/rancher/gke-operator
+    urls:
+    - assets/rancher-gke-operator/rancher-gke-operator-2.0.100-rc1+up2.0.1-rc1.tgz
+    version: 2.0.100-rc1+up2.0.1-rc1
```

Comment:
This needs to be regenerated, shouldn't be adding gke 2.0 stuff in here

Reply:
Removed.